### DISCUSSION TITLE
关于路网联通性打断

### DISCUSSION THREAD

**Discussion by ren-max11** [zdsjjtTLG/TrackIt#13] [General]
我在使用算法进行路网打断，联通修复时，为什么出现路网交叉，但是没有打断？下面是代码顺序，麻烦博主帮忙看看
![Snipaste_2024-12-20_17-17-31](https://github.com/user-attachments/assets/d8b99e8d-3d77-425f-bc5a-3df530f90809)
![Snipaste_2024-12-20_17-20-53](https://github.com/user-attachments/assets/df200c44-6cec-4057-ac73-fb3534d8ff3b)

**Comment by zdsjjtTLG** [comment]
这些地方本身就没有节点，都是一些折点，用联通性修复是不会生产节点的，你试试使用路网重塑

**Reply by ren-max11** [reply]
我这次使用了redivide_node,但是算法生成节点的方式似乎是按照距离生成节点，而不是在路网的交叉部分生成节点，这样导致联通性修复时，路网变得很奇怪。     如果把生成点的距离参数变小，会生成太多的点，有什么解决办法吗？
![2](https://github.com/user-attachments/assets/bb3be9fb-6c7f-4c79-a928-e06f8bc356b4)
![1](https://github.com/user-attachments/assets/4dee7cbd-ca31-4c62-89ac-87f1c002b9d3)

**Reply by zdsjjtTLG** [reply]
贴一下代码照片

**Reply by ren-max11** [reply]
![image](https://github.com/user-attachments/assets/9aa1d826-f02c-4a11-b975-ce04e6cb86c5)

**Reply by zdsjjtTLG** [reply]
那其实就是说在那个路的交叉点处连折点都没有，重塑路网就是按照线的折点把LINK打散。另外这个交点处连折点都没有的话，是不是可能不是平交，而是立交？你确认一下。

**Reply by zdsjjtTLG** [reply]
目前是没有提供“强行把没有折点的交点打断联通”的方法函数，因为很多跨线桥不应该去联通
